spec/rails_helper.rb

```diff
@@ -187,7 +187,7 @@ def headless_download_setup(driver)
   # config.filter_gems_from_backtrace("gem name")
 
   # https://medium.com/@velciov.vlad/retrying-flaky-tests-fae14de26c1b
-  config.default_retry_count = 2
+  config.default_retry_count = 3
```

Comment:
Some time related specs fail only on CI tests, increased retry to 3 times until we fix that flaky ones